lib/plugin.js

```diff
@@ -14,7 +14,7 @@ function ManifestPlugin(opts) {
   }, opts || {});
 }
 
-ManifestPlugin.prototype.getFileType = function(str) {
+ManifestPlugin.prototype.getFileType = function (str) {
```

Comment:
uhmm. i think i can not trust on idea IDEs for formatting using editorconfig rules. I will look on editorconfig for a rule to add such restriction. Otherwise other will incur in same error